scripts/supercloud/run_predicators_sidelining_experiments.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+START_SEED=456
+NUM_SEEDS=10
+FILE="scripts/supercloud/submit_supercloud_job.py"
+# Note: this script is meant to be run first, to completion, with
+# RUN_LOAD_EXPERIMENTS=false, then rerun with RUN_LOAD_EXPERIMENTS=true.
+RUN_LOAD_EXPERIMENTS=false
+NUM_TRAIN_TASKS="50"
+ALL_ENVS=(
+    "repeated_nextto_painting"
+    "screws"
+    "repeated_nextto_single_option"
+    "painting"
+)
+
```

Comment:
using the $COMMON_ARGS pattern from the other scripts makes it easier to see the diffs between the approaches